views/lifeCycle.erb

```diff
@@ -0,0 +1,99 @@
+    <style>
+        text {
+            font: 12px sans-serif;
+        }
+        svg {
+            display: block;
+        }
+        html, body, svg {
+            margin: 0px;
+            padding: 0px;
+            height: 100%;
+            width: 100%;
+        }
+    </style>
+</head>
+<body class='with-3d-shadow with-transitions'>
+<button onclick="expandLegend();">Expand/Contract Legend</button>
+    <script>
+        var expandLegend = function() {
+            var exp = chart.legend.expanded();
+            chart.legend.expanded(!exp);
+            chart.update();
+        }
+    </script>
+<svg id="chart1"></svg>
+
+<script>
+    var histcatexplong = [
```

Comment:
More magic data.  I'm guessing that like above it isn't "our" data, it's just dummy data for helping demonstrate your visualizations, but explaining it will help future you as well as other folks who maintain this code later. :smile: